<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1201</width>
      <height>771</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgWidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0.00568182, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255))}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>751</width>
       <height>101</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
font: 27pt &quot;Palatino Linotype&quot;;
padding:10px;
</string>
     </property>
     <property name="text">
      <string>Pacemaker Device Control Module</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>430</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>430</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
